### PR TITLE
Group ahelp relayed messages by round id

### PR DESCRIPTION
**Pull request by wrexbe**
Changes the username in the ahelp relay to have the round id, and clears the cached messages when the round id changes.
R: 1234| N: UserNameHere

In theory this will make it easier to see all the messages from the same round grouped together, and should help push older messages into the past.

### CODE REVIEW

File: Content.Server/Administration/BwoinkSystem.cs [hunk 1]
```diff
@@ -50,6 +53,13 @@ public override void Initialize()
             _config.OnValueChanged(CVars.GameHostName, OnServerNameChanged, true);
             _sawmill = IoCManager.Resolve<ILogManager>().GetSawmill("AHELP");
             _maxAdditionalChars = GenerateAHelpMessage("", "", true, true).Length + Header("").Length;
+
+            SubscribeLocalEvent<RoundStartingEvent>(RoundStarting);
+        }
+
+        private void RoundStarting(RoundStartingEvent ev)
+        {
+            _relayMessages.Clear();
```
Comment (PaulRitter) [inline Content.Server/Administration/BwoinkSystem.cs:62]: no. why? no.

**Review by PaulRitter — CHANGES_REQUESTED** [review]

File: Content.Server/Administration/BwoinkSystem.cs (hunk 1 above)
Comment (PaulRitter) [reply Content.Server/Administration/BwoinkSystem.cs:62]: this'll drop all unsent messages at the end of the round. why?

**Review by PaulRitter — CHANGES_REQUESTED** [review]
also you'd wanna add the roundid with the message to _relayMessages. rn its sending the roundid of when the message was sent to discord, not when it was sent ingame

**Comment by wrexbe** [comment]
Capture
But it works Paul, I just tried it

**Comment by PaulRitter** [comment]
nvm

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**